infra/modules/gcp-oauth-long-access-strategy-read/main.tf

```diff
@@ -0,0 +1,10 @@
+locals {
+  secret_id = "PSOXY_ACCESS_TOKEN_${var.function_name}"
+}
+
+# secret to hold a long-lived access token
+# previously created by module gcp-oauth-long-access-strategy
```

Comment:
then why not pass it as a variable? you don't need this at all

Reply:
yeah, removed. I misunderstood how this was supposed to work. I thought it needed more frequent runs by the customer